jwst/associations/lib/rules_level2_base.py

```diff
@@ -1451,10 +1451,21 @@ def get_opt_element(self):
         """
         item = self.direct_image.item
         opt_elems = []
-        for keys in [["filter", "band"], ["pupil", "grating"]]:
-            opt_elem = getattr_from_list_nofail(item, keys, _EMPTY)[1]
-            if opt_elem:
-                opt_elems.append(opt_elem)
+        inst = str(item.get("instrume", "Unknown"))
```

Comment:
I don't think Camel-case keyword value is common? I think traditionally it is all uppercase from the FITS days.

```suggestion
        inst = str(item.get("instrume", "UNKNOWN"))
```